docs/CPP_STYLE.md

```diff
@@ -191,15 +193,15 @@ v = w * (x + z);
 
 ## Tips and Hints
 
-- If you see outdated code which can be improved - DO IT NOW (but in the separate pull request)! Make this awesome world even more better! 
+- If you see outdated code which can be improved - DO IT NOW (but in a separate pull request)!
 - Your code should work at least on [mac|win|linux|android][x86|x86_64], [ios|android][armv7|arm64] architectures
```

Comment:
Does our code work on windows?

Reply:
We support x86 architecture as well.

Reply:
well, it says "at least"
Removed win, added x86